# Nikolaus Wagner (C) 2023
# nwagner@lincoln.ac.uk

# No arguments
---
std_msgs/Bool success